segmentstore/server/src/test/java/io/pravega/segmentstore/server/store/StreamSegmentServiceNoOpWriteOnlyTests.java

```diff
@@ -24,6 +25,7 @@
  * because user segment write operation is no-oped.
  */
 @Slf4j
+@Ignore
```

Comment:
Why?

Reply:
NoOp storage does not yet support ChunkedSegmentStorage (aka SLTS)

Reply:
Ignoring only relevant tests now